src/FieldWarning/Assets/Camera/SlidingCameraBehaviour.cs

```diff
@@ -154,14 +154,14 @@ private void Update()
         if (Input.GetMouseButton(2)) {
             RotateCamera();
         }
-        
+
     }
 
     private void LateUpdate()
     {
         var dx = _translateX < GetScaledPanSpeed() ? _translateX : GetScaledPanSpeed();
         var dz = _translateZ < GetScaledPanSpeed() ? _translateZ : GetScaledPanSpeed();
-
+        _targetPosition = transform.position;
```

Comment:
Why? This breaks the lerping.

Reply:
If the _targetPosition isnt updated then after clicking on the minimap and getting the cam teleported it will teleport right back since the target Position is still the old position. Also i can still move the cam aroung just as before, so i dont know whats breaking then.